### PR TITLE
Fix QICore-to-USCore-to-BaseFHIR mapping on index page; Rearrange

### PR DESCRIPTION
**Pull request by matthewtiller**
QDM-to-QICore mapping and add negation tables

### CODE REVIEW

File: pages/index.md [hunk 1]
```diff
@@ -218,6 +219,7 @@ any, and the underlying FHIR resources:
 | | [USCore Laboratory Result](http://hl7.org/fhir/us/core/StructureDefinition-us-core-observation-lab.html) | [Observation]({{site.data.fhir.path}}observation.html) |
 | | [USCore Pediatric BMI for Age](http://hl7.org/fhir/us/core/StructureDefinition-pediatric-bmi-for-age.html) | [Observation]({{site.data.fhir.path}}observation.html) |
 | | [USCore Pediatric Weight for Height](http://hl7.org/fhir/us/core/StructureDefinition-pediatric-weight-for-height.html) | [Observation]({{site.data.fhir.path}}observation.html) |
+| | [USCore Pulse Oximetry](http://hl7.org/fhir/us/core/STU3.1/StructureDefinition-us-core-pulse-oximetry.html) | [Observation]({{site.data.fhir.path}}observation.html) |
```
Comment (c-schuler) [inline pages/index.md]: Please remove version specifier from url:
http://hl7.org/fhir/us/core/StructureDefinition-us-core-pulse-oximetry.html

Comment (matthewtiller) [reply pages/index.md]:
```suggestion
| | [USCore Pulse Oximetry](http://hl7.org/fhir/us/core/StructureDefinition-us-core-pulse-oximetry.html) | [Observation]({{site.data.fhir.path}}observation.html) |
```

**Comment by matthewtiller** [comment]
GitHub was suggesting that I could make the change within the PR review itself, so I tried to do that. I think it worked.

File: pages/qdm-to-qicore.md [hunk 2]
```diff
@@ -1612,7 +1552,7 @@ indicating information about medications that have been dispensed.
...
 | relevant dateTime             | [MedicationDispense.whenHandedOver](StructureDefinition-qicore-medicationdispense-definitions.html#MedicationDispense.whenHandedOver)                                              | When provided to patient or representative                                                                                       |
-| relevant Period               | [MedicationDispense.extension:validityPeriod](StructureDefinition-qicore-medicationdispense-definitions.html#MedicationDispense.extension:validityPeriod)                          |                                                                                                                                  |
+| relevant Period               | [MedicationDispense.daysSupply](StructureDefinition-qicore-medicationdispense-definitions.html#MedicationDispense.daysSupply)                          |                                                                                                                                  |
```
Comment (c-schuler) [inline pages/qdm-to-qicore.md]: How does this work? daysSupply is a quantity, not a period.

File: pages/qdm-to-qicore.md [hunk 3]
```diff
@@ -1862,27 +1802,17 @@ To capture specific types of "Physical Exam, Recommended" as QICoreObservationNo
...
-    the care of persons with health problems that does not involve
-    direct physical contact with a patient. Examples include patient
-    education and therapeutic communication.
-
-* *Procedure* is an act whose immediate and primary outcome
-    (post-condition) is the alteration of the physical condition of the
-    subject. A *procedure* may be a surgery or other type of physical
-    manipulation of a person’s body in whole or in part for purposes of
-    making observations and diagnoses or providing treatment.
+QDM defines Procedure an act whose immediate and primary outcome
```
Comment (c-schuler) [inline pages/qdm-to-qicore.md]: Wording is a little confusing. Suggest changing to "QDM defines Procedure as an act ..."

File: pages/qdm-to-qicore.md [hunk 4]
```diff
@@ -1862,27 +1802,17 @@ To capture specific types of "Physical Exam, Recommended" as QICoreObservationNo
...
 #### 8.20.1 Procedure Vs Intervention
 
-FHIR references both of these concepts using the *Procedure* resource,
-specifically noting a process that involves verification of the
-patient's comprehension or to change the patient's mental state would be
-a Procedure. Therefore, both QDM *datatypes*, Procedure and Intervention
-are included in this section of the QDM to QI-Core mapping especially
-since all of the QDM attributes for each of these QDM *datatypes* are
-identical.
+FHIR references both of this concept using the Procedure resource, specifically
```
Comment (c-schuler) [inline pages/qdm-to-qicore.md]: Suggest changing to "FHIR references both of these concepts using ..."

File: pages/qdm-to-qicore.md [hunk 5]
```diff
@@ -1424,7 +1453,6 @@ profile address Immunization, Administered.
...
-|                              | [MedicationAdministration.dosage.rate\[x\]](StructureDefinition-qicore-medicationadministration-definitions.html#MedicationAdministration.dosage.rate[x])     | Simple Quantity                                                         |
 | route                        | [MedicationAdministration.dosage.route](StructureDefinition-qicore-medicationadministration-definitions.html#MedicationAdministration.dosage.route)           |                                                                         |
 | frequency                    | [MedicationAdministration.request](StructureDefinition-qicore-medicationadministration-definitions.html#MedicationAdministration.request)                     | Reference to original MedicationRequest with content about prescription |
 |                              | [MedicationAdministration.dosage.rate\[x\]](StructureDefinition-qicore-medicationadministration-definitions.html#medicationadministration.dosage.rate[x])     | Identifies the speed with which the medication was or will be introduced into the patient (e.g., infusion rate).                              |
```
Comment (c-schuler) [inline pages/qdm-to-qicore.md]: I don't think you introduced this broken link, but would you mind resolving it?
Should be: `StructureDefinition-qicore-medicationadministration-definitions.html#MedicationAdministration.dosage.rate[x]`

Comment (matthewtiller) [reply pages/qdm-to-qicore.md]: This one, I've seen it labeled as an error, but I don't know why. Is your link not exactly what is already there? The existing link actually works in a browser, too. I don't know what the problem is.

File: pages/qdm-to-qicore.md (hunk 4 above)
Comment (matthewtiller) [reply pages/qdm-to-qicore.md]: changed as suggested

File: pages/qdm-to-qicore.md (hunk 3 above)
Comment (matthewtiller) [reply pages/qdm-to-qicore.md]: changed as suggested

File: pages/qdm-to-qicore.md (hunk 2 above)
Comment (matthewtiller) [reply pages/qdm-to-qicore.md]: I've relayed the question to Floyd on this. This is actually part of a bigger conversation I'm having with Floyd. We may end up removing timing information from Medications, or fixing timing in another way.

Comment (matthewtiller) [reply pages/qdm-to-qicore.md]: ok, actually this will be changed to "N/A" in the next commit

File: pages/qdm-to-qicore.md (hunk 5 above)
Comment (c-schuler) [reply pages/qdm-to-qicore.md]: Yes, the link does work. However, the case is incorrect. It appears that the validator is case-sensitive. Rightly so, as there are web servers like IIS where urls are case-sensitive.

Current: `StructureDefinition-qicore-medicationadministration-definitions.html#medicationadministration.dosage.rate[x]`

Suggested: `StructureDefinition-qicore-medicationadministration-definitions.html#MedicationAdministration.dosage.rate[x]`

Comment (matthewtiller) [reply pages/qdm-to-qicore.md]: Resolved in latest commit

**Pull request merged**